spec/latest/core/index.html

```diff
@@ -630,6 +630,40 @@ <h3>Discovering a Hydra-powered Web API</h3>
       using HTTP Link Headers is typically not necessary as the concepts
       used in responses from the API will dereference to their
       documentation.</p>
+
+    <p>Another case would involve an <i>ApiDocumentation</i> discover from
+      another Api response. There is no restriction on emitting an HTTP
+      <i>Link</i> header on every Api response, thus every response could be
+      used to discover <i>ApiDocumentation</i>.</p>
+
+    <pre class="example nohighlight" data-transform="updateExample"
+         title="Retrieving a Hydra API documentation from another Api response">
+      <!--
+      GET /api/items HTTP/1.1
+      Host: api.example.com
+      Accept: application/ld+json
+
+      ====================================
+
+      HTTP/1.1 200 OK
+      ...
+      Content-Type: application/ld+json
+      ****Link: <http://api.example.com/doc/>; rel="http://www.w3.org/ns/hydra/core#apiDocumentation"****
+
+      {
+        "@context": "http://www.w3.org/ns/hydra/context.jsonld",
+        "@id": "http://api.example.com/api/items",
+        "title": "Items collection",
+        ...
+      }
+      -->
+    </pre>
+
+    <p class="note">It is possible to use a fallback behavior in case an
+      Api response does not present an HTTP <i>Link</i> header. In such a situation,
+      client may use <i>Host</i> HTTP header value that was used to receive a response
```

Comment:
I don't follow. Where is the `Host` header behaviour described?

Reply:
Ok, maybe I should rephrase it. In general, I wanted to enable a fallback behavior in a situation when call to `http://some.url/api/items` does not provide an _ApiDocumentation_ link, client could call `http://some.url/` to obtain it (where `some.url` is actually a `Host`)

Reply:
I don't think we should have fallback defined that way. `Host` will be greatly unreliable. In most cases it may not even be intended for dereferencing.

Just the more reason to have ApiDoc mandatory for every response.

Reply:
I've softened the Link header requirement with SHOULD